tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -38,10 +38,10 @@ Feature: full text search
     And user "Alice" has uploaded file with content "hello nepal" to "uploadFolder/file3.txt"
     And user "Alice" has created the following tags for file "uploadFolder/file1.txt" of the space "Personal":
       | tag1 |
-    And user "Alice" has created the following tags for file "uploadFolder/file2.txt" of the space "Personal":
-      | tag1 |
     And user "Alice" has shared folder "/uploadFolder" with user "Brian"
     And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    And user "Brian" has created the following tags for file "uploadFolder/file2.txt" of the space "Shares":
+      | tag1 |
     When user "Brian" searches for "Tags:tag1" using the WebDAV API
```

Comment:
is it necessary to create a tag here as well? Does the tag created by the `sharer` not work

Reply:
There was the separete scenario for 
- tag searches by sharee where tag is created by user `Alice`
- tag searches by sharee where tag is created by sharee `Brian`

Later on after the review suggestion of @saw-jan dai https://github.com/owncloud/ocis/pull/6659#discussion_r1250188148
the two scenario can be merged where it  covers scenario for tag searches by sharee  where tag is created by sharee `Brian` and user `Alice`

Reply:
yes, that makes sense but I think we should create a different tag for `Brian`? Maybe `Brian` creates `tag2` and Brian searches for `tag1` and then also `tag2` the resources should be there in both search results.

Reply:
okay they're tagging different files. I didn't notice that earlier